Changelog.md

```diff
@@ -2,6 +2,11 @@ OpenCore Changelog
 ==================
 #### v0.8.1
 - Improved `ExtendBTFeatureFlags` quirk on newer macOS versions, thx @lvs1974
+- ocvalidate: always print version compatibility info
```

Comment:
We have one README in ocvalidate specifically for tracking the changes of it. Thus, there is no need to duplicate it here.